atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1959,6 +1959,9 @@ v8::Local<v8::Value> WebContents::GetLastWebPreferences(
 }
 
 bool WebContents::IsRemoteModuleEnabled() const {
+  if (web_contents()->GetVisibleURL().SchemeIs("chrome-devtools")) {
```

Comment:
This disables it for `chrome-devtools` but the original code marked `chrome-extension:` as a safe protocol as well.  Should that be referenced here somewhere?

Reply:
I am disabling the remote module for the background script host process here
https://github.com/electron/electron/pull/16866/files#diff-8d85c651cb0fb8feef2c4694d52da2e3R95
that one is loading the `chrome-extensions:` URL

Reply:
we only have handle devtools explicitly, as those don't have webPreferences

Reply:
👍